examples/6tisch/simple-node/project-conf.h

```diff
@@ -70,11 +70,15 @@
 /*******************************************************/
 
 /* Logging */
+#ifndef CONTIKI_TARGET_Z1
+/* Use the default log level on Z1 to save ROM. */
 #define LOG_CONF_LEVEL_RPL                         LOG_LEVEL_WARN
 #define LOG_CONF_LEVEL_TCPIP                       LOG_LEVEL_WARN
 #define LOG_CONF_LEVEL_IPV6                        LOG_LEVEL_WARN
 #define LOG_CONF_LEVEL_6LOWPAN                     LOG_LEVEL_WARN
 #define LOG_CONF_LEVEL_MAC                         LOG_LEVEL_INFO
+#endif
+
 /* Do not enable LOG_CONF_LEVEL_FRAMER on SimpleLink,
    that will cause it to print from an interrupt context. */
 #ifndef CONTIKI_TARGET_SIMPLELINK
```

Comment:
If all other logs are disabled on Z1, extend this condition to include Z1 too (unless it's required for the Cooja tests).